website/static/js/project.js

```diff
@@ -13,6 +13,7 @@ var LogFeed = require('js/logFeed.js');
 
 var ctx = window.contextVars;
 var NodeActions = {}; // Namespace for NodeActions
+require('css/style.css');
```

Comment:
This shouldn't be necessary; IIRC, style.css is loaded with a `link` tag on every page.